containers/postgresql/docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+if [ "$ENV" == "dev" ]; then
+  # prepare-db
+fi
+
+exec postgres "$@"
```

Comment:
@xmnlab without `postgres` here, the entrypoint was breaking the container with the log: `/docker-entrypoint.sh: line 7: exec: -p: invalid option`. Any idea?